<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1052</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ordenes</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1081</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(230, 233, 236);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="labelOP">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(85, 0, 127);
text-decoration: underline;
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>OPERARIO:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSRV">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>20</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(85, 0, 127);
text-decoration: underline;
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>SERVICIO:</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
    </widget>
    <widget class="QLabel" name="labelHR">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>20</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(85, 0, 127);
text-decoration: underline;
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>HORA:</string>
     </property>
    </widget>
    <widget class="QLabel" name="nombreOP">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(85, 0, 127);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Carolina Colina Zamorano</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
    </widget>
    <widget class="QLabel" name="servicio">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>20</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(85, 0, 127);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>servicio</string>
     </property>
    </widget>
    <widget class="QLabel" name="hora">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>20</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(85, 0, 127);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>hora</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>950</x>
       <y>10</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <activeon>reload.png</activeon>
      </iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>1081</width>
      <height>561</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1051</width>
       <height>501</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="rowCount">
      <number>4</number>
     </property>
     <row/>
     <row/>
     <row/>
     <row/>
     <column>
      <property name="text">
       <string>MÁQUINA</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>MATERIAL</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>CARROS</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>UBICACIÓN</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>DISPONIBLES</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>CUBIERTAS</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>PENDIENTES</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>PRIORIDAD</string>
      </property>
     </column>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>500</y>
       <width>1051</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(230, 233, 236);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="btn_llevarMaterial">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>10</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:&quot;steelblue&quot;;
color: &quot;white&quot;;
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Llevar Material</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_buscarrMaterial">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>10</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:&quot;steelblue&quot;;
color: &quot;white&quot;;
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Buscar Material</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>tableWidget</sender>
   <signal>itemClicked(QTableWidgetItem*)</signal>
   <receiver>btn_llevarMaterial</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>809</x>
     <y>592</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
